README.md

```diff
@@ -37,6 +37,14 @@ conda install -c conda-forge kedro
 
 Our [Get Started guide](https://docs.kedro.org/en/stable/get_started/install.html) contains full installation instructions, and includes how to set up Python virtual environments.
 
+### Installation from source
+To install Kedro from source code, clone the repository and run:
+```
+git clone git@github.com:kedro-org/kedro.git
+cd kedro
+pip install .
+```
```

Comment:
Can I suggest that you split this into two, so the `git clone` command, then the text "Change to the `kedro` directory and enter the following: `pip install .`"